src/backend/distributed/sql/udfs/citus_extradata_container/latest.sql

```diff
@@ -0,0 +1,9 @@
+/* udfs/citus_extradata_container/9.2-4.sql */
+
+DROP FUNCTION IF EXISTS citus_extradata_container(INTERNAL);
+CREATE FUNCTION citus_extradata_container(INTERNAL)
```

Comment:
Maybe inline this in citus--9.3-1--9.2-4? I find it confusing that this is 9.2-4.sql when this is actually 7.0-1.